source/draw/DrawState.ooc

```diff
@@ -22,8 +20,12 @@ BlendMode: enum {
 }
 
 DrawState: cover {
+	// Used by all
 	target: Image = null
 	inputImage: Image = null
+	// Only used by the write function
+	_originReference := FloatPoint2D new(0.0f, 0.0f) // Start of text
+	// Only used by the draw function
```

Comment:
How about a table in markdown showing availability of settings for different drawing commands?